q2cli/commands.py

```diff
@@ -251,6 +255,11 @@ def handle_in_params(self, kwargs):
         cmd_fallback = self.cmd_config_handler.get_value(kwargs)
 
         verbose = self.verbose_handler.get_value(kwargs, fallback=cmd_fallback)
+        quiet = self.quiet_handler.get_value(kwargs, fallback=cmd_fallback)
+
+        if verbose and quiet:
+            raise ValueError('Unsure of how to be quiet and verbose at the '
```

Comment:
Can you use the standard way of error reporting in Click? i.e. `click.secho(err=True, fg='red', bold=True)` and `click.exit(1)`?

We really need a helper for this -- I noticed we have a few different ways of error handling in q2cli. But that's for another day...